dags/analyzer_helper/discord/discord_transform_raw_data.py

```diff
@@ -98,23 +93,23 @@ def transform(
                 if data.get("replied_user"):
                     interactions.append(
                         self.create_interaction(
-                            "reply", [data["replied_user"]], "emitter"
+                            data=data, name="reply", author=data["author"], engaged_users=[data["replied_user"]], interaction_type="emitter"
                         )
                     )
-                    receiver_interaction = self.create_receiver_interaction(
-                        data, "reply", data["replied_user"], data["author"]
+                    receiver_interaction = self.create_interaction(
+                        data=data, name="reply", author=data["author"], engaged_users=[data["replied_user"]], interaction_type="receiver"
```

Comment:
line 99 and 100 should be something like this

```python
self.create_interaction(
                        data=data, name="reply", author=data["replied_user"], engaged_users=[data["author"]], 
interaction_type="receiver")
```

basically,  in the case of replies the receiver of a reply is the person being replied to. So we're assigning the author to the person being replied to (a.k.a `data["replied_user"]`) and the engaged user would be the author of the message that replied to another person.